# Working with Protozoan Parasite Database Resources 2022 (Africa)
Learn how to use online resources for genomic-scale data analysis of protozoan parasites
This course aims to provide Africa-based biomedical researchers and scientists working on protozoan parasites with hands-on experience in genomic-scale data analysis; including genome browsers and comparison tools, primary data analysis, methods for data integration and resources for sophisticated data mining (including the VEuPathDB search strategy system).

Examples and exercises will be drawn primarily from the VEuPathDB.org resources focusing on protozoan parasites including Apicomplexa (i.e. Plasmodium, Toxoplasma & Cryptosporidium) and Kinetoplastida (ie. Trypanosoma & Leishmania). Applicants working on any organism included in these resources will benefit.

The intensive programme will combine hands-on exercises with lectures and discussions on genomics and bioinformatics techniques, along with seminars from distinguished regional guest speakers.

The course is free to attend and open to research scientists and clinicians/clinical scientists/healthcare professionals based in Africa working with protozoan parasitic organisms. Prior coding or bioinformatics knowledge is not required.

This is a virtual course, breakout rooms will be heavily used to ensure adequate teacher/student ratio and to foster active discussion. Lectures will be recorded and made available to students.

# Target audience

The course is open to scientists or healthcare professionals based in Africa working with protozoan parasitic organisms.

The programme will include lectures, discussions and practical computational sessions covering the following topics:

Visualisation of sequence features and next generation sequencing data using the Artemis genome browser and annotation tool
Genome browsing and comparative genomics (ACT and J/GBrowse)
Formulating complex biological queries using the VEuPathDB search strategy system
Web-based resources including the GeneDB.org and VEuPathDB.org and GalaxyIntegrating functional genomics data (RNA-seq, ChIP-seq, ChIP-chip, microarrays, proteomics, etc) with genome sequence data
Theoretical aspects of genome re-sequencing
Querying genetic diversity on a genome-wide scale
Metabolic pathways and pathway and GO enrichment analysis
Working with next-generation sequence data (mapping, variant calling, RNA-Seq etc) using both command-line and web-based (Galaxy) methods
Introduction to community curation tools
Participants are encouraged to bring their own data, to use in consultation and/or in-workshop analysis

# Learning Outcomes

After attending this course, participants should be able to:

Use state-of-the-art bioinformatics software including web-based resources as well as in a UNIX-like environment to explore parasite datasets.
Identify opportunities and limitations of the different methodologies in analysing parasite genomics data
Apply software and web-based resources to visualise and analyse next generation sequencing datasets for parasitology research.
Apply software and web-based services to browse genomes and to display genome comparisons.
Integrate and query large-scale genomic and functional genomic databases to answer biology-related research questions and develop testable hypotheses
 

# Prerequisites

Applicants should be Africa-based scientists or healthcare professionals working with protozoan parasitic organisms.

 # Instructors and Speakers
 
 Luisa Berná - Institut Pasteur de Montevideo, Uruguay
 Ulrike Böhme - Wellcome Sanger Institute, UK
 Omar Harb - University of Pennsylvania, USA
 Collins Misita - University of Ghana, Ghana
 Thomas Otto - University of Glasgow, UK
 Dinkorma Ouologuem Toure - Université des Science, Techniques et des Technologies de Bamako, Mali
 
 
